plugins/services/src/js/components/forms/NetworkingFormSection.js

```diff
@@ -137,15 +145,36 @@ class NetworkingFormSection extends mixin(StoreMixin) {
       address = `${hostname}${Networking.L4LB_ADDRESS}${port}`;
     }
 
+    const loadBalancerTooltipContent = (
```

Comment:
I would say; Not worth the effort. What we should do is create smaller components that we could use in both sections. That was a clear tradeoff we made when we were working on this feature.